src/app/utils/encodingUtils.ts

```diff
@@ -0,0 +1,3 @@
+export function isEncodingAvailable(cache: DetailedInfinispanCache): boolean {
+  return cache.encoding.key !== 'Empty' || cache.encoding.value !== 'Empty';
```

Comment:
@karesti let me know, if this okay, so I can update backport

Reply:
you can use the enum